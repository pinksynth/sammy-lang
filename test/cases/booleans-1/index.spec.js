const {
  expectCompiledEval,
  performSnapshotAssertions,
} = require("../../helpers")
performSnapshotAssertions("Variables", __dirname)

test("Booleans implementation", () => {
  expectCompiledEval(`false`).toBe(false)
  expectCompiledEval(`true`).toBe(true)
})
