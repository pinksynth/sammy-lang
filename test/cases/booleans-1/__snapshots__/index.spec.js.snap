// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ“¸ Variables 1`] = `
[
  {
    "columnNumberEnd": 5,
    "columnNumberStart": 1,
    "lineNumberEnd": 1,
    "lineNumberStart": 1,
    "tokenType": "BOOLEAN",
    "value": "true",
  },
  {
    "columnNumberEnd": 1,
    "columnNumberStart": 5,
    "lineNumberEnd": 2,
    "lineNumberStart": 1,
    "tokenType": "WHITESPACE",
    "value": "
",
  },
  {
    "columnNumberEnd": 6,
    "columnNumberStart": 1,
    "lineNumberEnd": 2,
    "lineNumberStart": 2,
    "tokenType": "BOOLEAN",
    "value": "false",
  },
]
`;

exports[`ðŸ“¸ Variables 2`] = `
{
  "children": [
    {
      "columnNumberStart": 1,
      "lineNumberStart": 1,
      "type": "LITERAL_BOOLEAN",
      "value": "true",
    },
    {
      "columnNumberStart": 1,
      "lineNumberStart": 2,
      "type": "LITERAL_BOOLEAN",
      "value": "false",
    },
  ],
  "type": "ROOT",
}
`;

exports[`ðŸ“¸ Variables 3`] = `"true;false"`;
